worker/src/Channel/ChannelRequest.cpp

```diff
@@ -83,7 +83,7 @@ namespace Channel
 		{ FBS::Request::Method::RTPOBSERVER_REMOVE_PRODUCER,                    "rtpObserver.removeProducer"                 },
 	};
 	// clang-format on
-
+	thread_local flatbuffers::FlatBufferBuilder ChannelRequest::bufferBuilder{};
```

Comment:
This must be moved above to a new "Static variables." section at the top as done in many other cpp files. Also ensure you are using tab rather than spaces.